Deep_Learning/HoneypotAI/honeypot/ssh_server.py

```
@@ -0,0 +1,202 @@
+                self._simulate_failed_login(client_socket)
+                return False
+                
+        except Exception as e:
+            self.logger.error(f"Error processing SSH connection: {e}")
+            return False
+    
+    def _simulate_ssh_handshake(self, client_socket: socket.socket):
+        """Simulate SSH protocol handshake"""
+        try:
+            # Receive key exchange init
+            key_exchange = client_socket.recv(1024)
```

Ruff (0.12.2)

Local variable `key_exchange` is assigned to but never used

Remove assignment to unused variable `key_exchange`

(F841)